config/vufind/Search2.ini

```diff
@@ -62,6 +62,7 @@ retain_filters_by_default = true
 ;default_filters[] = "format:Book"
 ;default_filters[] = "institution:MyInstitution"
 ;default_filters[] = "(format:Book AND institution:MyInstitution)"
+default_record_fields = *,score
```

Comment:
I'd like to encourage using quotes in ini values. Something like this feels much safer in quotes. :)

Reply:
Yes, good point. I did test that it works without quotes, but you're right that it feels better with them.